packages/core/src/parser/parser.ts

```diff
@@ -67,9 +68,17 @@ export interface CstResult {
 }
 
 /**
- * Label associated with a token that is only accepted in fault tolerant mode
+ * Additional token types
  */
-const faultTolerantToken = "FaultTolerantToken";
+enum AdditionalToken {
+    START_NEW_LINE = "StartNewLine",
+    FAULT_TOLERANT = "FaultTolerantToken"
```

Comment:
Yes
I mainly needed it, because the autocomplete executes the code, and then checks what is available when the correct position in the code is reached.
Thus, some errors in the code need to be automatically corrected to still be able to execute the code, mainly when you type a `.` and then want to see what's available.
Would it be better to introduce a lot of type inference?
Probably, however, this would be a lot of work and way out of the scope of this project sadly

Reply:
This token thus is only used in a specific parser mode, which is only used when doing autocomplete